Common/DataModel/EventSelection.h

```diff
@@ -78,36 +78,37 @@ namespace evsel
 // TODO bool arrays are not supported? Storing in int32 for the moment
 DECLARE_SOA_COLUMN(Alias, alias, int32_t[kNaliases]);
 DECLARE_SOA_COLUMN(Selection, selection, int32_t[kNsel]);
-DECLARE_SOA_COLUMN(BBV0A, bbV0A, bool);                 //! Beam-beam time in V0A
-DECLARE_SOA_COLUMN(BBV0C, bbV0C, bool);                 //! Beam-beam time in V0C
-DECLARE_SOA_COLUMN(BGV0A, bgV0A, bool);                 //! Beam-gas time in V0A
-DECLARE_SOA_COLUMN(BGV0C, bgV0C, bool);                 //! Beam-gas time in V0C
-DECLARE_SOA_COLUMN(BBFDA, bbFDA, bool);                 //! Beam-beam time in FDA
-DECLARE_SOA_COLUMN(BBFDC, bbFDC, bool);                 //! Beam-beam time in FDC
-DECLARE_SOA_COLUMN(BGFDA, bgFDA, bool);                 //! Beam-gas time in FDA
-DECLARE_SOA_COLUMN(BGFDC, bgFDC, bool);                 //! Beam-gas time in FDC
-DECLARE_SOA_COLUMN(MultRingV0A, multRingV0A, float[5]); //! V0A multiplicity per ring (4 rings in run2, 5 rings in run3)
-DECLARE_SOA_COLUMN(MultRingV0C, multRingV0C, float[4]); //! V0C multiplicity per ring (4 rings in run2)
-DECLARE_SOA_COLUMN(SpdClusters, spdClusters, uint32_t); //! Number of SPD clusters in two layers
-DECLARE_SOA_COLUMN(NTracklets, nTracklets, int);        //! Tracklet multiplicity
-DECLARE_SOA_COLUMN(Sel7, sel7, bool);                   //! Event selection decision based on V0A & V0C
-DECLARE_SOA_COLUMN(Sel8, sel8, bool);                   //! Event selection decision based on TVX
-DECLARE_SOA_COLUMN(FoundFT0, foundFT0, int32_t);        //! FT0 entry index in FT0s table (-1 if doesn't exist)
-DECLARE_SOA_COLUMN(FoundFV0, foundFV0, int32_t);        //! FV0 entry index in FV0A table (-1 if doesn't exist)
+DECLARE_SOA_COLUMN(BBV0A, bbV0A, bool);                                     //! Beam-beam time in V0A
+DECLARE_SOA_COLUMN(BBV0C, bbV0C, bool);                                     //! Beam-beam time in V0C
+DECLARE_SOA_COLUMN(BGV0A, bgV0A, bool);                                     //! Beam-gas time in V0A
+DECLARE_SOA_COLUMN(BGV0C, bgV0C, bool);                                     //! Beam-gas time in V0C
+DECLARE_SOA_COLUMN(BBFDA, bbFDA, bool);                                     //! Beam-beam time in FDA
+DECLARE_SOA_COLUMN(BBFDC, bbFDC, bool);                                     //! Beam-beam time in FDC
+DECLARE_SOA_COLUMN(BGFDA, bgFDA, bool);                                     //! Beam-gas time in FDA
+DECLARE_SOA_COLUMN(BGFDC, bgFDC, bool);                                     //! Beam-gas time in FDC
+DECLARE_SOA_COLUMN(MultRingV0A, multRingV0A, float[5]);                     //! V0A multiplicity per ring (4 rings in run2, 5 rings in run3)
+DECLARE_SOA_COLUMN(MultRingV0C, multRingV0C, float[4]);                     //! V0C multiplicity per ring (4 rings in run2)
+DECLARE_SOA_COLUMN(SpdClusters, spdClusters, uint32_t);                     //! Number of SPD clusters in two layers
+DECLARE_SOA_COLUMN(NTracklets, nTracklets, int);                            //! Tracklet multiplicity
```

Comment:
This is not important for this PR, but aren't 16 bit enough here?

Reply:
Yes, good point. we have up to 6000 tracklets in Run2 Pb-Pb so 16 bit should be enough. I will adjust it in the next iteration